workers/unity/Packages/com.improbable.gdk.tools/.CodeGenerator/GdkCodeGenerator/src/Generation/Model/UnityTypeMappings.cs

```diff
@@ -1,45 +1,29 @@
-        private const int MetadataComponentId = 53;
-        private const int PositionComponentId = 54;
-        private const int PersistenceComponentId = 55;
-
-        public static readonly HashSet<int> WellKnownComponents = new HashSet<int>
-        {
-            EntityAclComponentId,
-            MetadataComponentId,
-            PositionComponentId,
-            PersistenceComponentId
-        };
-
         public static readonly Dictionary<string, string> BuiltInSchemaTypeToUnityNativeType =
             new Dictionary<string, string>
             {
-                { BuiltInTypeConstants.builtInDouble, "double" },
-                { BuiltInTypeConstants.builtInFloat, "float" },
-                { BuiltInTypeConstants.builtInInt32, "int" },
-                { BuiltInTypeConstants.builtInInt64, "long" },
-                { BuiltInTypeConstants.builtInUint32, "uint" },
-                { BuiltInTypeConstants.builtInUint64, "ulong" },
-                { BuiltInTypeConstants.builtInSint32, "int" },
-                { BuiltInTypeConstants.builtInSint64, "long" },
-                { BuiltInTypeConstants.builtInFixed32, "uint" },
-                { BuiltInTypeConstants.builtInFixed64, "ulong" },
-                { BuiltInTypeConstants.builtInSfixed32, "int" },
-                { BuiltInTypeConstants.builtInSfixed64, "long" },
-                { BuiltInTypeConstants.builtInBool, "BlittableBool" },
-                { BuiltInTypeConstants.builtInString, "string" },
-                { BuiltInTypeConstants.builtInBytes, "byte[]" },
-                { BuiltInTypeConstants.builtInEntityId, "global::Improbable.Gdk.Core.EntityId" },
-                { BuiltInTypeConstants.builtInCoordinates, "global::Improbable.Coordinates" },
-                { BuiltInTypeConstants.builtInVector3d, "global::Improbable.Vector3d" },
-                { BuiltInTypeConstants.builtInVector3f, "global::Improbable.Vector3f" }
+                { BuiltInSchemaTypes.BuiltInDouble, "double" },
+                { BuiltInSchemaTypes.BuiltInFloat, "float" },
+                { BuiltInSchemaTypes.BuiltInInt32, "int" },
+                { BuiltInSchemaTypes.BuiltInInt64, "long" },
+                { BuiltInSchemaTypes.BuiltInUint32, "uint" },
+                { BuiltInSchemaTypes.BuiltInUint64, "ulong" },
+                { BuiltInSchemaTypes.BuiltInSint32, "int" },
+                { BuiltInSchemaTypes.BuiltInSint64, "long" },
+                { BuiltInSchemaTypes.BuiltInFixed32, "uint" },
+                { BuiltInSchemaTypes.BuiltInFixed64, "ulong" },
+                { BuiltInSchemaTypes.BuiltInSfixed32, "int" },
+                { BuiltInSchemaTypes.BuiltInSfixed64, "long" },
+                { BuiltInSchemaTypes.BuiltInBool, "BlittableBool" },
+                { BuiltInSchemaTypes.BuiltInString, "string" },
+                { BuiltInSchemaTypes.BuiltInBytes, "byte[]" },
+                { BuiltInSchemaTypes.BuiltInEntityId, "global::Improbable.Gdk.Core.EntityId" }
```

Comment:
what happened to the vector and coordinate types?

Reply:
They are handled by the normal generation process (i.e. - they have an `Identifier` and are in the lookup table)